### DISCUSSION TITLE
Handle ansi control codes for color

### DISCUSSION THREAD

**Discussion by imsjrhowa** [Almenon/AREPL-vscode#457] [Ideas]
Would be great to see color come thru..

# ANSI escape codes for colors
class Style:
    RESET = '\x1b[0m'
    BOLD = '\x1b[1m'

class Color:
    RESET = '\x1b[0m'
    BOLD = '\x1b[1m'
    RED = '\x1b[91m'
    GREEN = '\x1b[92m'
    YELLOW = '\x1b[93m'
    BLUE = '\x1b[94m'
    MAGENTA = '\x1b[95m'
    CYAN = '\x1b[96m'

print(f"{Style.BOLD}{Color.RED}This is bold red text.{Color.RESET}")

**Comment by Almenon** [comment]
Yeah, this would be nice. Someone else requested this feature too, so I'll close this discussion and add your +1 to https://github.com/Almenon/AREPL-vscode/issues/437
